llvm/lib/Target/AArch64/AArch64.td

```diff
@@ -1325,6 +1352,10 @@ def ProcessorFeatures {
                                  FeatureMatMulInt8, FeatureBF16, FeatureAM,
                                  FeatureMTE, FeatureETE, FeatureSVE2BitPerm,
                                  FeatureFP16FML];
+  list<SubtargetFeature> A520 = [HasV9_2aOps, FeatureNEON, FeaturePerfMon,
```

Comment:
`FeatureNeon`, `FeatureMatMulInt8`, `FeatureBF16` and `FeatureFineGrainedTraps` are already enabled by default on Armv9.2-A. Can they be removed from this list to avoid the redundancy?